utilities/test_suite/HOST/Tensor_host_audio.cpp

```diff
@@ -0,0 +1,444 @@
+            printf("\ncase %d is not supported\n", testCase);
+
+        return -1;
+    }
+
+    // Initialize tensor descriptors
+    RpptDesc srcDesc, dstDesc;
+    RpptDescPtr srcDescPtr, dstDescPtr;
+    srcDescPtr = &srcDesc;
+    dstDescPtr = &dstDesc;
+
+    // Set src/dst data types in tensor descriptors
+    if (inputBitDepth == 2)
+    {
+        srcDescPtr->dataType = RpptDataType::F32;
+        dstDescPtr->dataType = RpptDataType::F32;
+    }
+
+    // Other initializations
+    int missingFuncFlag = 0;
+    int i = 0, j = 0, fileCnt = 0;
+    int maxChannels = 0;
+    int maxSrcWidth = 0, maxSrcHeight = 0;
+    int maxDstWidth = 0, maxDstHeight = 0;
+    unsigned long long count = 0;
+    unsigned long long iBufferSize = 0;
+    unsigned long long oBufferSize = 0;
+    static int noOfAudioFiles = 0;
+
+    // String ops on function name
+    char src1[1000];
+    strcpy(src1, src);
+    strcat(src1, "/");
+
+    string func = funcName;
+    cout << "\nRunning " << func;
+
+    // Get number of audio files
+    vector<string> audioNames;
+    vector<string> audioFilePath;
+
+    search_files_recursive(src, audioNames, audioFilePath, ".wav");
+    noOfAudioFiles = audioNames.size();
+
+    if (noOfAudioFiles < batchSize || ((noOfAudioFiles % batchSize) != 0))
+    {
+        replicate_last_file_to_fill_batch(audioFilePath[noOfAudioFiles - 1], audioFilePath, audioNames, audioNames[noOfAudioFiles - 1], noOfAudioFiles, batchSize);
+        noOfAudioFiles = audioNames.size();
+    }
+
+    // Initialize the AudioPatch for source
+    Rpp32s *srcLengthTensor = (Rpp32s *) calloc(noOfAudioFiles, sizeof(Rpp32s));
+    Rpp32s *channelsTensor = (Rpp32s *) calloc(noOfAudioFiles, sizeof(Rpp32s));
+    RpptImagePatch *srcDims = (RpptImagePatch *) calloc(noOfAudioFiles, sizeof(RpptImagePatch));
+    RpptImagePatch *dstDims = (RpptImagePatch *) calloc(noOfAudioFiles, sizeof(RpptImagePatch));
+
+    // Set Height as 1 for src, dst
+    maxSrcHeight = 1;
+    maxDstHeight = 1;
+
+    for (int cnt = 0; cnt < noOfAudioFiles ; cnt++)
+    {
+        SNDFILE	*infile;
+        SF_INFO sfinfo;
+        int	readcount;
+
+        //The SF_INFO struct must be initialized before using it
+        memset (&sfinfo, 0, sizeof (sfinfo));
+        if (!(infile = sf_open (audioFilePath[cnt].c_str(), SFM_READ, &sfinfo)))
+        {
+            sf_close (infile);
+            continue;
+        }
+
+        srcLengthTensor[count] = sfinfo.frames;
+        channelsTensor[count] = sfinfo.channels;
+
+        srcDims[count].width = sfinfo.frames;
+        dstDims[count].width = sfinfo.frames;
+        srcDims[count].height = 1;
+        dstDims[count].height = 1;
+
+        maxSrcWidth = std::max(maxSrcWidth, srcLengthTensor[count]);
+        maxDstWidth = std::max(maxDstWidth, srcLengthTensor[count]);
+        maxChannels = std::max(maxChannels, channelsTensor[count]);
+
+        // Close input
+        sf_close (infile);
+        count++;
+
+    }
+
+
```

Comment:
Please go over the code quickly and remove any double line spaces. Also from L263-L278, lets compact by removing too many empty lines - similar elsewhere

Reply:
Done
Somehow it is not showing in github UI though